cosmos/joltify_1729.json

```diff
@@ -0,0 +1,56 @@
+{
+  "rpc": "https://rpc.joltify.io",
+  "rest": "https://lcd.joltify.io",
+  "nodeProvider": {
+    "name": "Joltify",
+    "email": "contact@joltify.io",
+    "website":"https://joltify.io"
+  },
+  "chainId": "joltify_1729-1",
+  "chainName": "Joltify",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/joltify_1729/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "JOLT",
+    "coinMinimalDenom": "ujolt",
+    "coinDecimals": 6,
+    "coinGeckoId": "joltify",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/joltify_1729/ujolt.png"
+  },
+  "walletUrl": "https://app.joltify.io/assets",
```

Comment:
Please remove this line since walletUrl was deprecated.

Reply:
walletUrl removed